apps/web/components/App.tsx

```diff
@@ -52,9 +55,20 @@ export default function App({
   email: string; // required
   tos?: string;
   privacy?: string;
+  variant: string;
 }) {
   const { t } = useLocale();
 
+  const { isSuccess, isLoading, data } = trpc.useQuery(["viewer.integrations"]);
+  let appCredentials: { credentialIds: number[] | undefined } | undefined;
+
+  if (data) {
+    let apps = data[variant as keyof typeof data];
+    appCredentials = apps.items.find((item: { type: any }) => item.type === type);
+  }
```

Comment:
Experiencing some typescript errors so had to wrap it in an if statement

@zomars I saw your comment about making this into an endpoint. Since other parts require `data` is something we can address later or should I do it now?

Reply:
From what I can see we're only checking that endpoint to see if we have a credentialID for that app, why we don't add an input for the variant and just return true/false or the credentials id's?